Werewolf for Telegram/Werewolf Control/Handlers/UpdateHandler.cs

```diff
@@ -791,6 +816,12 @@ internal static void HandleCallback(CallbackQuery query)
                         return;
                     }
 
+                    if (args[0] == "setgrouptopic_cmd")
+                    {
+                        Commands.SetGroupTopicCallback(query);
+                        return;
+                    }
+
```

Comment:
As stated before, this should be dropped in favor of simply 2 commands for setting and unsetting.